xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-filters/xwiki-platform-notifications-filters-default/src/main/java/org/xwiki/notifications/filters/internal/livedata/custom/NotificationCustomFiltersLiveDataConfigurationProvider.java

```diff
@@ -146,18 +140,6 @@ public LiveDataConfiguration get()
         entryDescriptor.setIdProperty(ID_FIELD);
         meta.setEntryDescriptor(entryDescriptor);
 
-        LiveDataActionDescriptor deleteAction = new LiveDataActionDescriptor();
-        deleteAction.setName(this.l10n.getTranslationPlain("liveData.action.delete"));
-        deleteAction.setId(DELETE);
-        deleteAction.setAllowProperty(DOC_HAS_DELETE_FIELD);
```

Comment:
You're sure it's ok to remove this ? The standard action already check for `doc_hasdelete` field to enable the action?

Reply:
Hum indeed, I'd missed that part. Thanks for catching it.
So we still need to define the action descriptor, but only with the minimal entries we need to override + the id for the merge.
I'll update the PR soon.